tree/ntuple/v7/test/ntuple.cxx

```diff
@@ -1,913 +0,0 @@
-#include <ROOT/RColumnModel.hxx>
-#include <ROOT/RDataFrame.hxx>
-#include <ROOT/RNTuple.hxx>
-#include <ROOT/RNTupleDescriptor.hxx>
-#include <ROOT/RNTupleDS.hxx>
-#include <ROOT/RNTupleModel.hxx>
-#include <ROOT/RNTupleOptions.hxx>
-#include <ROOT/RPageStorage.hxx>
-#include <ROOT/RPageStorageFile.hxx>
-#include <ROOT/RVec.hxx>
-
-#include <TClass.h>
-#include <TFile.h>
-#include <TRandom3.h>
-
-#include "gtest/gtest.h"
-
-#include "CustomStruct.hxx"
-
-#include <array>
-#include <cstdio>
-#include <exception>
-#include <memory>
-#include <string>
-#include <utility>
-#if __cplusplus >= 201703L
-#include <variant>
-#endif
-
-using DescriptorId_t = ROOT::Experimental::DescriptorId_t;
-using EColumnType = ROOT::Experimental::EColumnType;
-using ENTupleStructure = ROOT::Experimental::ENTupleStructure;
-using NTupleSize_t = ROOT::Experimental::NTupleSize_t;
-using RColumnModel = ROOT::Experimental::RColumnModel;
-using RNTupleDescriptor = ROOT::Experimental::RNTupleDescriptor;
-using RNTupleDescriptorBuilder = ROOT::Experimental::RNTupleDescriptorBuilder;
-using RNTupleReader = ROOT::Experimental::RNTupleReader;
-using RNTupleReadOptions = ROOT::Experimental::RNTupleReadOptions;
-using RNTupleWriter = ROOT::Experimental::RNTupleWriter;
-using RNTupleWriteOptions = ROOT::Experimental::RNTupleWriteOptions;
-using RNTupleModel = ROOT::Experimental::RNTupleModel;
-using RNTupleVersion = ROOT::Experimental::RNTupleVersion;
-using RPageSink = ROOT::Experimental::Detail::RPageSink;
-using RPageSinkFile = ROOT::Experimental::Detail::RPageSinkFile;
-using RPageSource = ROOT::Experimental::Detail::RPageSource;
-using RPageSourceFile = ROOT::Experimental::Detail::RPageSourceFile;
-using RFieldBase = ROOT::Experimental::Detail::RFieldBase;
-
-namespace {
-
-/**
- * An RAII wrapper around an open temporary file on disk. It cleans up the guarded file when the wrapper object
- * goes out of scope.
- */
-class FileRaii {
-private:
-   std::string fPath;
-public:
-   explicit FileRaii(const std::string &path) : fPath(path) { }
-   FileRaii(const FileRaii&) = delete;
-   FileRaii& operator=(const FileRaii&) = delete;
-   ~FileRaii() { std::remove(fPath.c_str()); }
-   std::string GetPath() const { return fPath; }
-};
-
-} // anonymous namespace
-
-
-TEST(RNTuple, Basics)
```

Comment:
Two very short tests were removed all together: 
RNTuple Basics and RNTuple Capture further down.